v2/bigquery-to-bigtable/src/main/java/com/google/cloud/teleport/v2/templates/BigQueryToBigtable.java

```diff
@@ -127,13 +158,23 @@ public static void main(String[] args) {
 
     BigQueryToBigtableOptions options =
         PipelineOptionsFactory.fromArgs(args).withValidation().as(BigQueryToBigtableOptions.class);
-    CloudBigtableTableConfiguration bigtableTableConfig =
+    CloudBigtableTableConfiguration.Builder builderBigtableTableConfig =
         new CloudBigtableTableConfiguration.Builder()
             .withProjectId(options.getBigtableWriteProjectId())
             .withInstanceId(options.getBigtableWriteInstanceId())
             .withAppProfileId(options.getBigtableWriteAppProfile())
-            .withTableId(options.getBigtableWriteTableId())
-            .build();
+            .withTableId(options.getBigtableWriteTableId());
+
+    if (options.getBigtableLatencyMsTarget() != null) {
+      builderBigtableTableConfig // I need to confirm which BigtableOptions are right here
+          .withConfiguration(BigtableOptionsFactory.BIGTABLE_BUFFERED_MUTATOR_ENABLE_THROTTLING, "true")
+          .withConfiguration(BigtableOptionsFactory.BIGTABLE_BUFFERED_MUTATOR_THROTTLING_THRESHOLD_MILLIS, options.getBigtableLatencyMsTarget())
+          .withConfiguration(BigtableOptionsFactory.BIGTABLE_BULK_MAX_ROW_KEY_COUNT, options.getBigtableMaxRowKeyCount())
+          .withConfiguration(BigtableOptionsFactory.BIGTABLE_BULK_MAX_REQUEST_SIZE_BYTES, options.getBigtableBulkMaxRequestSizeBytes());
```

Comment:
why do these depend on latency target being present?

Reply:
It shouldn't be, I changed it so each of them have an if statement.